syntaxes/omt.tmLanguage.json

```diff
@@ -223,6 +224,36 @@
                 }
             ]
         },
+        "_merge-predicates-def": {
+            "patterns": [
+                { "include": "#comments" },
+                {
+                    "begin": "(?:\\G|^) +(subjects|predicates|when): +",
+                    "beginCaptures": {
+                        "1": { "name": "keyword.control.omt" }
+                    },
+                    "end": "$",
+                    "contentName": "meta.embedded.inline.odt",
+                    "patterns": [{ "include": "source.odt" }]
+                },
+                {
+                    "match": "(?:\\G|^) +(from): *(parent)?(.*)$",
```

Comment:
In the future (from) gets more options right? If those options are added we have to change tis again. Can we make this support the other options too?

Reply:
I could do that, but then I'd have to add the `use` option as well, and that doesn't work at all yet. Does save doing a PR for that tiny change in the plugin.